mods/page_save.js

```diff
@@ -0,0 +1,117 @@
+'use strict';
+
+
+/**
+ * page_save module
+ *
+ * Sends the wikitext of a page to the MW API for saving
```

Comment:
I'm wondering if we should integrate this with the Parsoid module. There is already wikitext retrieval functionality in there, as well as some of the revision info logic. The html-based save process will be tightly coupled with wikitext saving as well.

What do you think?